src/components/encoding-pane/property-editor.tsx

```diff
@@ -67,7 +68,33 @@ export class PropertyEditorBase extends React.PureComponent<PropertyEditorProps,
       });
     }
   }
+
+  private parseFormDataResult(result: string) {
+    const {fieldDef, prop, nestedProp} = this.props;
+    const reg = /\s*,\s*/; // regex for parsing comma delimited strings
+    if (prop === 'scale') {
+      if (nestedProp === 'range') {
+        const range = result.split(reg);
+        if (isContinuous(fieldDef) && range.length !== 2) {
+          throw new Error('Invalid format for range. Must follow format: Min Number, Max Number');
+        }
+        return result === '' ? '?' : range;
+      } else if (nestedProp === 'domain') {
+        const domain = result.split(reg);
+        if (fieldDef.type === ExpandedType.QUANTITATIVE && domain.length !== 2) {
+          throw new Error('Invalid format for domain. Must follow format: Min Number, Max Number');
+        } else if (fieldDef.type === ExpandedType.TEMPORAL) {
+          // TODO: Not supported yet
+          throw new Error('Voyager does not currently support temporal domain values');
+        }
+        return result === '' ? '?' : domain;
+      }
+    }
+
+    // if form data is empty, default to auto suggested view, which is ? in compass
```

Comment:
I think this logic won't work because Compass doesn't recommend properties of scale, axis, legend.   The right thing to is do set to `undefined` and it will use Vega-Lite's default.  

Thus, you should revise all logic that deals with this `'?'`.